APIAssignment1.py

```diff
@@ -0,0 +1,101 @@
+import json
+import requests
+import csv
```

Comment:
order of lib import

Reply:
https://www.python.org/dev/peps/pep-0008/#:~:text=Imports%20are%20always%20put%20at,Standard%20library%20imports.